### PR TITLE
1359-solvo-biotechnology

### PR DESCRIPTION
**Pull request by ahiuchingau**

### CODE REVIEW

**Comment by ahiuchingau** [comment]
closes #410

File: protocols/1359-solvo-biotechnology/serial_dilution.ot2.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        special_mix_identifiers: str='A, B, 10',
+        special_mix_height: float=7,
+        dilution_transfer_volume: float=20,
+        deep_plate_transfer_volume: float=5):
+
+    # serial dilution
+    for row in plate.rows():
+        p50.transfer(
+            dilution_transfer_volume,
+            row.wells('12', to='2'),
```
Comment (Laura-Danielle) [inline protocols/1359-solvo-biotechnology/serial_dilution.ot2.py]: uhh, does this actually work?

I was under the impression that you can only index with specified well name. alternatively you can do:
```
row[11:1]
```

File: protocols/1359-solvo-biotechnology/serial_dilution.ot2.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            row.wells('11', to='1'),
```
Comment (Laura-Danielle) [inline protocols/1359-solvo-biotechnology/serial_dilution.ot2.py]: And here would be:

```
row[10:0]
```

File: protocols/1359-solvo-biotechnology/serial_dilution.ot2.py (hunk 1 above)
Comment (ahiuchingau) [reply protocols/1359-solvo-biotechnology/serial_dilution.ot2.py]: I uploaded it to the app and it seems to work for me

**Pull request merged**